handel/src/services/amazonmq/index.ts

```diff
@@ -52,14 +52,14 @@ const AMAZON_MQ_PORTS = [
     61619 // WSS
 ];
 
-// This is pretty hacky...better add a good crypto random string soon
+// There are 36 charatcers because the uuid in the old code returned a 36 character username
```

Comment:
This service doesn't update anyway so the 36-character requirement doesn't need to be met. Let's remove these comments and make the username shorter; how about 12 characters? Let's leave the passwords at the longer length though.